### PR TITLE
Leaf/alan/dynamic search results component

### PR DESCRIPTION
**Pull request by alinz22**
A list component retrieving and displaying data from RMP and showcasing it on a react extension.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,5 +1,39 @@
 import React, { ReactElement } from 'react';
+import ListPage from './ListItem';
+
+const personArray = [
```
Comment (drjonah) [inline src/App.tsx:4]: test data must be removed eventually and replaced with a function to fetch backend data

File: src/ListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+interface person {
```
Comment (drjonah) [inline src/ListItem.tsx:3]: These are the attributes that were storing so you might need to add them to the interface to be displayed:

firstName
lastName
avgDifficulty
avgRating
numRatings
wouldTakeAgainPercent

Comment (drjonah) [reply src/ListItem.tsx:3]: id and legacyId are also available but I don't think that these will be shown on front end

**Review by drjonah — APPROVED** [review]
looks good, the comments arent urgent but could help.

**Review by Leaversa — APPROVED** [review]

**Pull request merged**